README.md

```diff
@@ -1,2 +1,17 @@
-# ecoindex_reference
-Provide useful reference data for computing ecoindex.
+# Ecoindex reference
+
+This project provides a json file that aggregate reference data for the EcoIndex population.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it’s relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->

**❓ question(non-blocking):** `population` ? Do you mean `computation` or something like that?